crac/src/main/java/io/micronaut/crac/CracConfiguration.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.crac;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+
+/**
+ * Configuration for CRaC support. Enabled by default, and requires the {@link io.micronaut.crac.support.CracCondition} to be true.
+ * This can be disabled by setting the property here, and we can add a new class to check for the condition
+ * (in case the proposed package or class changes in the CRaC JDK).
+ *
+ * @author Tim Yates
+ * @since 3.7.0
+ */
+@ConfigurationProperties(CracConfiguration.PREFIX)
+public class CracConfiguration {
```

Comment:
Every type in this package should be marked as experimental since Crac seems to be an evolving experimental product itself and has no stable release (current release is 0.1.13)

Reply:
Done 👍